app/api/dao/mentorship_relation.py

```diff
@@ -1,11 +1,11 @@
-from datetime import datetime, timedelta
+is not Nonefrom datetime import datetime, timedelta
```

Comment:
Please do basic sanity checks and run your code before pushing it.

Reply:
It got pushed by mistake